src/test/java/org/apache/commons/text/similarity/LevenshteinDistanceTest.java

```diff
@@ -49,98 +46,112 @@ public void testGetLevenshteinDistance_StringString() {
 
     @Test(expected = IllegalArgumentException.class)
     public void testGetLevenshteinDistance_NullString() throws Exception {
+        LevenshteinDistance distance = new LevenshteinDistance();
+
         distance.compare("a", null);
     }
 
     @Test(expected = IllegalArgumentException.class)
     public void testGetLevenshteinDistance_StringNull() throws Exception {
+        LevenshteinDistance distance = new LevenshteinDistance();
+
         distance.compare(null, "a");
     }
 
     @Test
     public void testGetLevenshteinDistance_StringStringInt() {
+        Map<Integer, LevenshteinDistance> distances = new TreeMap<Integer, LevenshteinDistance>();
```

Comment:
Better implement this as a paramterized test using the (Parameterized test runner)[https://github.com/junit-team/junit/blob/master/src/main/java/org/junit/runners/Parameterized.java]. It would have to be moved to a new test class.